hometask1/hometask1_1.js

```diff
@@ -0,0 +1,6 @@
+var stdin = process.openStdin();
+
+stdin.addListener("data", function(data) {
```

Comment:
Since you're not utilising "this" key word inside this callback - consider using the arrow function syntax.